src/nemos/basis.py

```diff
@@ -771,34 +769,73 @@ def evaluate_on_grid(self, n_samples: int) -> Tuple[NDArray, NDArray]:
         return super().evaluate_on_grid(n_samples)
 
 
-class RaisedCosineBasis(Basis, abc.ABC):
-    def __init__(self, n_basis_funcs: int) -> None:
+class RaisedCosineBasisLinear(Basis):
+    """Represent linearly-spaced raised cosine basis functions.
+
+    This implementation is based on the cosine bumps used by Pillow et al.[$^1$](references)
```

Comment:
```suggestion
    This implementation is based on the cosine bumps used by Pillow et al.[$^1$](#references)
```

Reply:
Also, does the Pillow paper ever actually use this basis function?

Reply:
yep, it's eqn (5) in the paper